src/Tizen.NUI/src/public/Visuals/VisualAnimator.cs

```diff
@@ -163,6 +163,7 @@ protected override void ComposingPropertyMap()
             _transition.Add("animator", new PropertyValue(_animator));
 
             _outputVisualMap = _transition;
+            base.ComposingPropertyMap();
```

Comment:
it seems that new logic is added. could you confirm this?
this is very important because no bug or no behavior change is not allowed.